src/getting-started.md

```diff
@@ -21,8 +21,12 @@ chapter on how to build and run the compiler](./building/how-to-build-and-run.md
 
 ## Asking Questions
 
-If you have questions, please make a post on [internals.rust-lang.org][internals] or
-hop on the [Rust Discord server][rust-discord] or [Rust Zulip server][rust-zulip].
+If you have questions, please make a post on the [Rust Zulip server][rust-zulip] or
+[internals.rust-lang.org][internals]. If you are contributing to Rustup, be aware they are not on
+Zulip - you can ask questions in `#wg-rustup` [on Discord][rust-discord].
+See [the Community page][community] on the official website for more resources.
```

Comment:
[imo] This isn't a strong opinion so feel free to ignore, but we can find a team/WG's chat space and repository on https://www.rust-lang.org/governance. I guess it's the _most_ up-to-date information resource and it makes some sense to add a link here.

Reply:
Great idea, thanks! I'd forgotten that the chat platforms are linked on the governance page.